config/environments/development.rb

```diff
@@ -14,11 +14,17 @@
 
   # Enable/disable caching. By default caching is disabled.
   # Run rails dev:cache to toggle caching.
-  
-  
+
+  # User will be timed out after 90 minutes regardless of activity
+  config.session_expiry_in_minutes = 90
```

Comment:
this can live in `config/application.rb` and we only add this to the `/config/environments/<env>.rb` when we need to overwrite the default... and that might be enough (i.e. no need for env variables)...